packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -145,6 +145,7 @@ const Popup: React.FC<PopupProps> &
     offset,
     pointing,
     position,
+    positionFixed,
```

Comment:
do we need to do the same for `MenuButton`?

Reply:
Good catch, added 👍